pkg/builder/builder.go

```diff
@@ -838,6 +890,9 @@ func makeServerSecurityContext(vdb *vapi.VerticaDB) *corev1.SecurityContext {
 // makeContainers creates the list of containers to include in the pod spec.
 func makeContainers(vdb *vapi.VerticaDB, sc *vapi.Subcluster) []corev1.Container {
 	cnts := []corev1.Container{makeServerContainer(vdb, sc)}
+	if vdb.IsSideCarDeploymentEnabled() {
```

Comment:
I think we may want the NMA to be the first container. The first container is the default one if you do a `kubectl exec` without specifying a container. This will come in handy if you want to run scrutinize. As it should be the nma container that you run this in.

Reply:
I will have nma be the first container.